tools/depends/target/Makefile

```diff
@@ -81,6 +81,12 @@ else ifeq ($(TARGET_PLATFORM),aml)
   DEPENDS += libamcodec
 endif
 
+ifneq ($(TARGET_PLATFORM),)
```

Comment:
this is superfluous, the condition below includes this

Reply:
no it isn't, `TARGET_PLATFORM` can be empty (android, osx, linux X11)
`$(filter $(TARGET_PLATFORM),raspberry-pi aml gbm)` is empty if `TARGET_PLATFORM` is empty.
Then `ifeq ($(TARGET_PLATFORM),$(filter $(TARGET_PLATFORM),raspberry-pi aml gbm))` would evaluate to true.